src/LAMMPS.jl

```diff
@@ -1,12 +1,69 @@
 module LAMMPS
 import MPI
 include("api.jl")
+import .API: _LMP_STYLE_CONST, LMP_STYLE_GLOBAL, LMP_STYLE_ATOM, LMP_STYLE_LOCAL
 
 export LMP, command, get_natoms, extract_atom, extract_compute, extract_global,
-       gather, scatter!, group_to_atom_ids, get_category_ids
+       extract_setting, gather, scatter!, group_to_atom_ids, get_category_ids,
+       extract_variable,
+
+       LAMMPS_NONE,
+       LAMMPS_INT,
+       LAMMPS_INT_2D,
+       LAMMPS_DOUBLE,
+       LAMMPS_DOUBLE_2D,
+       LAMMPS_INT64,
+       LAMMPS_INT64_2D,
+       LAMMPS_STRING,
+
+       TYPE_SCALAR,
+       TYPE_VECTOR,
+       TYPE_ARRAY,
+       SIZE_VECTOR,
+       SIZE_ROWS,
+       SIZE_COLS,
+
+       VAR_EQUAL,
+       VAR_ATOM,
+       VAR_VECTOR,
+       VAR_STRING,
+
+       LMP_STYLE_GLOBAL,
+       LMP_STYLE_ATOM,
+       LMP_STYLE_LOCAL
```

Comment:
I would almost consider not exporting them.

And then name then `NONE` instead of `LAMMPS_NONE` and `STYLE_GLOBAL` instead of `LMP_STYLE_GLOBAL`.
Then it's consitent in user code as `LAMMPS.NONE` and `LAMMPS.STYLE_GLOBAL`